### ISSUE TITLE
Volume Icon

### ISSUE THREAD

**Issue by MoonPhoenix83** [kyleneideck/BackgroundMusic#442]
Could you release a version with the tweaked Big Sur Volume Icon alongside the existing icons?

**Comment by kyleneideck** [comment]
Good idea, but unfortunately not quite as simple as you would think. I recreated the volume icon images instead of having the app try to use the ones from the system. I guess I was thinking that those image files could be moved/renamed in a macOS update and then Background Music wouldn't be able to find them.

It would be nice if it tried to use the system images and then used the other ones if it couldn't find them. Or we could just update the images. They're generated from this LaTeX file if you're interested: https://github.com/kyleneideck/BackgroundMusic/blob/master/Images/VolumeIcons.tex

There's some instructions for it in the header comments.

**Comment by schwedenmut** [comment]
Yesterday I went ahead and used the original ones provided by macOS - but it was quite hacky achieved by just copying the Volume[0-3].pdf files to the corresponding Folder in BGMApp Folder before build.
I will take a look into that and hopefully provide a small PR for that.

**Comment by MoonPhoenix83** [comment]
Yes, please do! I tried to edit the tex files but I know nothing about xcode or terminal so I couldn't build it.

**Comment by schwedenmut** [comment]
Just got a hand on a Big Sur enabled Macbook and your thoughts @kyleneideck aboute icons moved away seems quite right. The folder for the Volume.menu entry isn't available anymore. Probably due to switching to the Control Center, who knows.

**Comment by kyleneideck** [comment]
Yeah, probably. Maybe in `/System/Library/CoreServices/ControlCenter.app/Contents/Resources/Assets.car`?
